File: my/src/main/java/com/littlecorgi/my/logic/model/Student.java
package com.littlecorgi.my.logic.model;

/**
 * 学生实体类
 *
 * @author littlecorgi 2021/05/03
 */
@lombok.NoArgsConstructor
@lombok.Data
public class Student {

    private Integer status;
    private String msg;
    private String errorMsg;
    private DataBean data;

    /**
     *
     */
    @lombok.NoArgsConstructor
    @lombok.Data
    public static class DataBean {
        private long id;
        private String name;
        private String email;
        private String password;
        private String phone;
        private String avatar;
        private String picture;
    }
}

File: my/src/main/java/com/littlecorgi/my/logic/model/Student.kt
package com.littlecorgi.my.logic.model

import java.io.Serializable

/**
 * 学生用户信息 - 数据类
 *
 * @author littlecorgi-twk 2021/06/10
 */
data class Student(
    var id: Long? = null,
    var avatar: String? = null,
    var email: String? = null,
    var name: String? = null,
    var password: String? = null,
    var phone: String? = null,
    var picture: String? = null
) : Serializable {
    companion object {
        private const val serialVersionUID = 5990939387657230607L
    }
}
